operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -114,6 +114,10 @@ type AdmissionControlComponentSpec struct {
 	ListenOnEvents *bool `json:"listenOnEvents,omitempty"`
 
 	// Should inline scanning be performed on previously unscanned images during a deployments admission review.
+    //
```

Comment:
Personally, as an end-user, I would actually prefer the description with proper newlines like this:
```
    // Should inline scanning be performed on previously unscanned images during a deployments admission review.
    //
    //
    // Valid values are:
    //
    // - "DoNotScanInline" (default): to skip scanning the image when processing the admission request
    //
    // - "ScanIfMissing": if the scan results for the image are missing
```

But I completely understand that it makes the actual code less readable. (This is where I really wish controller-gen/kubebuilder would make newlines easier or fix it in a better way.)